models/public/face-recognition-resnet100-arcface/model.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  Deep face recognition net with ResNet100 backbone and Arcface loss <https://arxiv.org/abs/1801.07698>
+task_type: face_recognition
+files:
+  - name: model-r100-arcface-ms1m-refine-v2.zip
+    sha256: eab0e5bcca81c070f56ffb11a1c32d7b2e574a8bb94f32590e7b48b355c13d1f
+    source: https://www.dropbox.com/s/tj96fsm6t6rq8ye/model-r100-arcface-ms1m-refine-v2.zip?dl=1
+    output: "face_recognition/insightface/mxnet/resnet100/"
```

Comment:
The `output` key is no longer supported (and it was never supported in _this_ position).